lib/Common/CommonDefines.h

```diff
@@ -102,6 +102,7 @@
 // xplat-todo: revisit these features
 #ifdef _WIN32
 #define ENABLE_INTL_OBJECT                          // Intl support
+#define ENABLE_BUILTIN_OBJECT                       // Built In functions support
```

Comment:
>ENABLE_BUILTIN_OBJECT [](start = 8, length = 21)

Is there going to be a Builtin object? Certainly not one exposed to the user though. Probably best to rename this to something like ENABLE_JS_BUILTINS